test/e2e/lib/jest/environment-fail-fast.js

```diff
@@ -0,0 +1,17 @@
+const JestEnvironmentNode = require( 'jest-environment-node' );
```

Comment:
This is used to implement failfast (abort current spec if it fails, don't touch other specs). Inspired by https://github.com/facebook/jest/issues/6527#issuecomment-760092817